### DISCUSSION TITLE
Forms and Multilanguages

### DISCUSSION THREAD

**Discussion by netwaretcs** [statamic/cms#6199] [Q&A]
Hello, Can anyone tell me if it is possible to use the 'trans' tag to translate form labels?

**Comment by duncanmcclean** [comment]
It should be - have you tried?

**Comment by netwaretcs** [comment]
I've tried but I didn't find any documentation.

`                    {{ fields }}
                    <div class="p-2">
                        <label>{{ display }}</label>
                        <div class="p-1">{{ field }}</div>
                    </div>
                    {{ /fields }}
`

How to use "trans" tag with {{ display }} and, for example, in the label option of {{ field }}?

**Reply by duncanmcclean** [reply]
Try `{{ trans :key="display" }}`

Taken from the [form page builder component](https://github.com/studio1902/statamic-peak/blob/main/resources/views/page_builder/_form.antlers.html#L44) in the Peak Starter Kit
